scvi/model/base/_base_model.py

```diff
@@ -112,9 +109,39 @@ def get_anndata_manager(
 
         return cls.manager_store[adata_uuid]
 
+    def get_from_registry(
+        self,
+        adata: AnnData,
+        registry_key: str,
+    ) -> np.ndarray:
```

Comment:
this actually also can return pandas dataframe, like if protein expression is a df